### PR TITLE
CURL Examples for Monitoring Service

### PR DESCRIPTION
**Pull request by dbrian57**
What is says on the tin! I verified each example using my account. Note: the JSON payloads for the POST and PUT requests require single quotes instead of double quotes, i.e. that formatting was intentional. Thanks, API team!

### CODE REVIEW

File: specification/resources/monitoring/examples/curl/create_alert.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+lang: cURL
+source: |-
+  curl -X POST \
+    -H "Content-Type: application/json" \
+    -H "Authorization: Bearer $DIGITALOCEAN_TOKEN" \
+    "https://api.digitalocean.com/v2/monitoring/alerts"
+    --data '<JSON-payload>'
```
Comment (scotchneat) [inline specification/resources/monitoring/examples/curl/create_alert.yml:7]: Unless the value for the `--data` arg was intentionally left as `<JSON-payload>`, I'd prefer to include a working example where possible. Something like:

```suggestion
    --data '{"alerts":{"email":["alerts@example.com"],"compare":"GreaterThan","description":"CPU Alert","enabled":true,"entities":["12345678"],"tags":["droplet_tag"],"type":"v1/insights/droplet/cpu","value":80,"window":"5m"}'
```

File: specification/resources/monitoring/examples/curl/create_alert.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+lang: cURL
+source: |-
+  curl -X POST \
+    -H "Content-Type: application/json" \
+    -H "Authorization: Bearer $DIGITALOCEAN_TOKEN" \
+    "https://api.digitalocean.com/v2/monitoring/alerts"
```
Comment (scotchneat) [inline specification/resources/monitoring/examples/curl/create_alert.yml:6]: This is missing the backslash after the URL value

```suggestion
    "https://api.digitalocean.com/v2/monitoring/alerts" \
```

File: specification/resources/monitoring/examples/curl/update_alert.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+lang: cURL
+source: |-
+  curl -X PUT \
+    -H "Content-Type: application/json" \
+    -H "Authorization: Bearer $DIGITALOCEAN_TOKEN" \
+    "https://api.digitalocean.com/v2/monitoring/alerts/78b3da62-27e5-49ba-ac70-5db0b5935c64"
```
Comment (scotchneat) [inline specification/resources/monitoring/examples/curl/update_alert.yml:6]: This also needs a backslash after the URL. 

```suggestion
    "https://api.digitalocean.com/v2/monitoring/alerts/78b3da62-27e5-49ba-ac70-5db0b5935c64" \
```

File: specification/resources/monitoring/examples/curl/update_alert.yml [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+    --data '<JSON-payload>'
```
Comment (scotchneat) [inline specification/resources/monitoring/examples/curl/update_alert.yml:7]: Same question/suggestion here for the `<JSON-payload>`

```suggestion
    --data ''{"alerts":{"email":["alerts@exmaple.com"],"slack":[{"channel":"Production Alerts","url":"https://hooks.slack.com/services/T1234567/AAAAAAAA/ZZZZZZ"}]},"compare":"GreaterThan","description":"CPU Alert","enabled":true,"entities":["12345678"],"tags":["droplet_tag"],"type":"v1/insights/droplet/cpu","value":80,"window":"5m"}''
```

**Review by scotchneat — CHANGES_REQUESTED** [review]
Thanks, @dbrian57! I added some suggestions and questions inline.

**Comment by dbrian57** [comment]
@scotchneat Thanks Cesar!

**Review by scotchneat — APPROVED** [review]
Awesome. LGMT 👍🏽

**Pull request merged**